crates/runtime/tests/runtime_tests.rs

```diff
@@ -247,98 +212,57 @@ fn memory_runtime_call_func_not_found() {
 }
 
 #[test]
-fn memory_runtime_spawn_without_gas() {
-    let mut runtime = memory_runtime();
-
-    // 1) `Deploy Template`
-    let code_version = 0;
-    let deployer = Address::of("deployer").into();
-    let layout: FixedLayout = vec![Address::len() as u32].into();
-    let ctors = vec!["initialize".to_string()];
-
-    let bytes = testing::build_template(
-        code_version,
-        "My Template",
-        layout.clone(),
-        &ctors,
-        (&include_bytes!("wasm/runtime_calldata.wasm")[..]).into(),
-    );
-
-    let receipt = runtime.deploy(&bytes, &deployer, Gas::new());
-    assert!(receipt.success);
-
-    let template_addr = receipt.addr.unwrap();
-
-    // 2) `Spawn Account`
-    let name = "My Account";
-    let ctor = "initialize";
-    let calldata = vec![];
-    let creator = Address::of("creator").into();
-    let bytes = testing::build_spawn(&template_addr, name, ctor, &calldata);
-    let receipt = runtime.spawn(&bytes, &creator, Gas::with(0));
-
-    assert!(matches!(receipt.error.unwrap(), RuntimeError::OOG));
-}
```

Comment:
What happened to this one?

Reply:
https://github.com/spacemeshos/svm/issues/313